integration-libs/opf/b2b-checkout/components/opf-b2b-checkout-payment-and-review/opf-b2b-checkout-payment-and-review.component.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChangeDetectionStrategy,
+  Component,
+  inject,
+  OnInit,
+} from '@angular/core';
+import { UntypedFormBuilder } from '@angular/forms';
+import { Cart, PaymentType } from '@spartacus/cart/base/root';
+import { CheckoutPaymentTypeFacade } from '@spartacus/checkout/b2b/root';
+import { CmsService } from '@spartacus/core';
+import {
+  OpfBaseFacade,
+  OpfMetadataStoreService,
+} from '@spartacus/opf/base/root';
+import { Observable, take, map, filter, combineLatest } from 'rxjs';
+import { Card } from '@spartacus/storefront';
+import { CheckoutStepType } from '@spartacus/checkout/base/root';
+import { OpfCheckoutPaymentAndReviewComponent } from '@spartacus/opf/checkout/components';
+
+@Component({
+  selector: 'cx-opf-b2b-checkout-payment-and-review',
+  templateUrl: './opf-b2b-checkout-payment-and-review.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: false,
+})
+export class OpfB2bCheckoutPaymentAndReviewComponent
+  extends OpfCheckoutPaymentAndReviewComponent
+  implements OnInit
+{
+  protected fb = inject(UntypedFormBuilder);
+  protected opfMetadataStoreService = inject(OpfMetadataStoreService);
+  protected cmsService = inject(CmsService);
+  protected checkoutPaymentTypeFacade = inject(CheckoutPaymentTypeFacade);
+  protected opfBaseFacade = inject(OpfBaseFacade);
```

Comment:
These are injected in the parent class.  Is it necessary to duplicate here?

Reply:
They are not needed I removed them.